packages/curve-ui-kit/src/themes/stories/ToggleButtons.stories.tsx

```diff
@@ -0,0 +1,151 @@
+/* eslint-disable react-hooks/rules-of-hooks */
+import { useState } from 'storybook/internal/preview-api' // Intentionally, can't use React's useState: https://github.com/storybookjs/storybook/issues/29189
+import type { Meta, StoryObj } from '@storybook/react'
+import ToggleButton from '@mui/material/ToggleButton'
+import ToggleButtonGroup from '@mui/material/ToggleButtonGroup'
+
+import FormatAlignLeftIcon from '@mui/icons-material/FormatAlignLeft'
+import FormatAlignCenterIcon from '@mui/icons-material/FormatAlignCenter'
+import FormatAlignRightIcon from '@mui/icons-material/FormatAlignRight'
+import FormatAlignJustifyIcon from '@mui/icons-material/FormatAlignJustify'
+
+import FormatBoldIcon from '@mui/icons-material/FormatBold'
+import FormatItalicIcon from '@mui/icons-material/FormatItalic'
+import FormatUnderlinedIcon from '@mui/icons-material/FormatUnderlined'
+import FormatColorFillIcon from '@mui/icons-material/FormatColorFill'
+
+import CheckIcon from '@mui/icons-material/Check'
+
+const meta: Meta<typeof ToggleButtonGroup> = {
+  title: 'UI Kit/Primitives/ToggleButton',
+  component: ToggleButtonGroup,
+  argTypes: {
+    size: {
+      control: 'select',
+      options: ['extraSmall', 'small', 'medium', 'extraSmallSquare', 'smallSquare', 'mediumSquare'],
```

Comment:
Do we really need this many sizes?
Afaik all buttons are square so I don't understand the difference

Reply:
The togglebuttons are rectangular with only a min width, most noticable with the text variant in the storybook and Figma. The block padding is also slightly larger than the inline padding.

The square sizes are really square with equal block and inline padding, fixed (responsive) width and hidden overflow. This variant isn't on the btn-toggle Figma page, but it's used in the user profile one.